sdk-http-vertx/src/main/java/dev/restate/sdk/http/vertx/RequestHttpServerHandler.java

```diff
@@ -86,6 +91,24 @@ public void handle(HttpServerRequest request) {
       return;
     }
 
+    // check protocol version
+    final String protocolVersionString = request.getHeader(CONTENT_TYPE);
+
+    final Protocol.ServiceProtocolVersion serviceProtocolVersion =
+        ServiceProtocol.parseServiceProtocolVersion(protocolVersionString);
+
+    if (!ServiceProtocol.is_supported(serviceProtocolVersion)) {
+      request
+          .response()
+          .setStatusCode(UNSUPPORTED_MEDIA_TYPE.code())
+          .setStatusMessage(
+              String.format(
+                  "Service endpoint does not support the service protocol version '%s'.",
+                  protocolVersionString))
```

Comment:
Remove this, see my comment below about reason phrases.

Reply:
👍